packages/gatsby/src/bootstrap/load-config/index.ts

```diff
@@ -53,14 +53,21 @@ export async function loadConfig({
       if (unfitFlagMessage !== ``) {
         reporter.warn(unfitFlagMessage)
       }
+
+      // Print out message.
+      if (message !== ``) {
+        reporter.info(message)
+      }
+
       //  set process.env for each flag
       enabledConfigFlags.forEach(flag => {
         process.env[flag.env] = `true`
       })
 
-      // Print out message.
-      if (message !== ``) {
-        reporter.info(message)
+      if (process.env.GATSBY_SLICES && process.env.GATSBY_PARTIAL_HYDRATION) {
+        delete process.env.GATSBY_SLICES
+
+        reporter.warn(`SLICES is inactive when PARTIAL_HYDRATION is enabled.`)
```

Comment:
this is temporary - it doesn't look greatest, as we still print regular flags messages and then there is this warning - I didn't want to waste time to add "mutually exclussive" flags generic handling given that we hope to have this resolved next week anyway

Reply:
Update here, this is still "temporary" but it will be postpone a bit further to get base for partial hydration in good spot before worrying about interop.